composer.json

```diff
@@ -9,7 +9,8 @@
         "vendor-dir": "administrator/components/com_patchtester/vendor"
     },
     "require": {
-        "php": "^5.3.10|^7.0"
+        "php": "^7.0",
```

Comment:
@mbabker This is why the build fails.

Reply:
You could not be more wrong right now if you tried.  Have you actually reviewed the CI logs?  I have, there is nothing in https://travis-ci.org/joomla-extensions/patchtester/builds/585002926 (latest build as of this comment) which says the build is failing because you changed the PHP version constraint.  Don’t just assume things when the data is at your fingertips.